lib/rubygems.rb

```diff
@@ -1348,6 +1348,7 @@ def clear_default_specs
   autoload :DependencyList,     File.expand_path('rubygems/dependency_list', __dir__)
   autoload :Installer,          File.expand_path('rubygems/installer', __dir__)
   autoload :Licenses,           File.expand_path('rubygems/util/licenses', __dir__)
+  autoload :NameTuple,          File.expand_path("rubygems/name_tuple", __dir__)
```

Comment:
@olleolleolle use single quotations 😄

Reply:
It was me who proposed that code :flushed:.

Reply:
There, force-pushed a change!

Reply:
Thanks! I think we'll move to double quotes when we unify the code style for bundler & rubygems, but for now it's best to keep the surrounding style :+1:.